### PR TITLE
chore: add language identfiers to android docs

### PR DESCRIPTION
**Pull request by brnhensley**
Add syntax highlighting to the mobile android docs, and one otel doc.

I can't figure out what might be the identifier for a dex config, it looks like maybe it doesn't have any syntactic highlighting options

### CODE REVIEW

**Comment by zuluecho9** [comment]
@brnhensley Brian, did you want us to look into something for that question you had?

**Comment by brnhensley** [comment]
Hey @zuluecho9 feel free to look into it, but I didn't make this a draft for that reason. Looking over official android docs they don't use any syntax highlighting for their dex code examples so maybe that's just the way it is. 

I wanted to wait for the build to see if `gradle` works as expected, and I miiight add another few docs to this if I have a slow afternoon.

**Comment by brnhensley** [comment]
Hey @zuluecho9, one of the things I tried to fix on [the otel doc](https://docswebsitedevelop-brnhensleydocswebsitepatc39278.gatsbyjs.io/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/opentelemetry-tutorial-java/) is eluding me. For some reason the closing triple-backticks is showing up as text in some of these examples, can you see why?

image
image
image

**Comment by brnhensley** [comment]
figures out the weird triple backtick problem this is ready for review now.

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps.mdx [hunk 1]
```diff
@@ -25,21 +25,21 @@ To add support for [ProGuard](http://proguard.sourceforge.net/manual/usage.html)
...
-   ```
+    ```
+    -keep class com.newrelic.** { *; }
+    -dontwarn com.newrelic.**
+    -keepattributes Exceptions, Signature, InnerClasses, LineNumberTable, SourceFile, EnclosingMethod
+
+    ##
+    ## NewRelic Gradle plugin 7.x may require the following additions:
+    ##
+    # Retain generic signatures of TypeToken and its subclasses if 8 version 3.0 full-mode (https://r8.googlesource.com/r8/+/refs/heads/master/compatibility-faq.md#r8-full-mode) is enabled.
```
Comment (rhetoric101) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps.mdx:36]: @brnhensley should we keep the "R" in "R8"? This seems less clear without that.

**Review by rhetoric101 — APPROVED** [review]
@brnhensley thanks for making all of these formatting improvements! I just had one question about R8.

As far as Dex, this is a reference to DexGuard for Android, no? In any case, I don't see any sign on the Prism site that Dex or DexGuard are supported.

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps.mdx (hunk 1 above)
Comment (brnhensley) [reply src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps.mdx]: I don't recall removing that intentionally, I think that's a typo. I just pushed a commit to correct this and make it a little nicer to look at too without losing the helpful link

**Comment by brnhensley** [comment]
> As far as Dex, this is a reference to DexGuard for Android, no? In any case, I don't see any sign on the Prism site that Dex or DexGuard are supported.

I believe you're correct @rhetoric101, I don't know much about this specifically but I did a little digging to understand the filetypes involved. Going off this link, `.dex` are what android compiles into, and dexguard is related in some way to that.
- https://stackoverflow.com/questions/7750448/what-are-dex-files-in-android

That said I found examples of dex on the android docs and none had syntax highlighting. Since google is good about that in other docs I'm assuming there isn't really a syntax highlighting option for dex in general
- https://source.android.com/docs/core/runtime/dex-format

**Review by rhetoric101 — APPROVED** [review]
@brnhensley thanks for the clarification! This looks good.

**Pull request merged**